### PR TITLE
Fix a typo  relating to the period length.

### PR DESCRIPTION
**Pull request by dwjohnston**

### CODE REVIEW

**Comment by dwjohnston** [comment]
I signed it!

File: src/blog/math-random.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ MWC1616 uses little memory and is pretty fast to compute, but unfortunately offe
 - The more significant upper half of the result is almost entirely dependent on the value of state0. The period length would be at most 232, but instead of few large permutation cycles, there are many short ones. With a badly chosen initial state, the cycle length could be less than 40 million.
```
Comment (mathiasbynens) [inline src/blog/math-random.md]: `232` should be `2<sup>32</sup>` (same on the line above, and for `252` as well)

File: src/blog/math-random.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ MWC1616 uses little memory and is pretty fast to compute, but unfortunately offe
...
 - It fails many statistical tests in the TestU01 suite.
 
-This has been [pointed out](https://medium.com/@betable/tifu-by-using-math-random-f1c308c4fd9d) to us, and having understood the problem and after some research, we decided to reimplement `Math.random` based on an algorithm called [xorshift128+](http://vigna.di.unimi.it/ftp/papers/xorshiftplus.pdf). It uses 128 bits of internal state, has a period length of 2128 - 1, and passes all tests from the TestU01 suite.
```
Comment (mathiasbynens) [inline src/blog/math-random.md]: - Let's use `2<sup>128</sup> - 1`.
- Let's fix the other issues above as well.

File: src/blog/math-random.md (hunk 1 above)
Comment (mathiasbynens) [reply src/blog/math-random.md]: Same for `2n` which should be `2<sup>n</sup>`

**Pull request merged**